tests/dsl/src/test/java/org/mule/test/extension/dsl/DeclarationBasedDslElementModelSerializerTestCase.java

```diff
@@ -60,6 +60,13 @@ private void createAppDeclaration() {
         .withConfig(db.newConfiguration("config")
             .withRefName("dbConfig")
             .withConnection(db.newConnection("derby-connection")
+                .withParameter("poolingProfile", newObjectValue()
+                    .withParameter("maxPoolSize", "10")
+                    .build())
+                .withParameter("connectionProperties", newObjectValue()
```

Comment:
test for "declaring a Map objectType", fixes => https://www.mulesoft.org/jira/browse/MULE-11565